docs/development/environment/index.md

```diff
@@ -38,6 +38,16 @@ line must be added to `www/web/sites/default/settings.php`:
 
     $settings['file_private_path'] = '/opt/drupal/web/sites/default/private/files';
 
+Additionally, create the folder `/opt/drupal/web/sites/default/private/`
+
+Set the correct user and permissions:
+
+Folder ownership and group should match the web server user. If you are using the farmOS Docker image (running Apache), this will be `www-data`.
```

Comment:
Can you break all lines that are longer than 80 characters for consistency with other docs? That's a convention we follow.

Reply:
A couple of questions:
do you want these small changes in separate commits?
what would you set as a commit message?
thx!

Reply:
Ultimately it will probably make sense to "[squash](https://stackoverflow.com/questions/35703556/what-does-it-mean-to-squash-commits-in-git)" all the commits from this branch into 1, so adding extra commits like "fix whitespace" is fine. I can squash 'em right before merging into the main `2.x` branch. Or, if you want to `git commit --amend` and force push the one commit that's fine too!